grudge/discretization.py

```diff
@@ -482,28 +533,49 @@ def _all_faces_volume_connection(self):
 
     @property
     def dim(self):
+        """Return the topological dimension."""
         return self._volume_discr.dim
 
     @property
     def ambient_dim(self):
+        """Return the dimension of the ambient space."""
         return self._volume_discr.ambient_dim
 
     @property
     def real_dtype(self):
+        """Return the data type used for real-valued arithmetic."""
         return self._volume_discr.real_dtype
 
     @property
     def complex_dtype(self):
+        """Return the data type used for complex-valued arithmetic."""
         return self._volume_discr.complex_dtype
 
     @property
     def mesh(self):
+        """Return a :class:`meshmode.mesh.Mesh` over which the discretization
+        collection is built.
+        """
         return self._volume_discr.mesh
 
     def empty(self, array_context: ArrayContext, dtype=None):
+        """Return an empty :class:`~meshmode.dof_array.DOFArray`.
```

Comment:
Mention that it's on `DD_VOLUME`?

Reply:
Good call! https://github.com/inducer/grudge/commit/52c7318c777efebf2c559986f0f7b6a429898e89